hq/app/AppComponents.scala

```diff
@@ -135,9 +141,18 @@ class AppComponents(context: Context)
     environment,
     cacheService
   )
+
+  val iamDynamoDbTableName: String = Config.getIamDynamoTableName(configuration) match {
+    case Some(tableName) => tableName
+    case None =>
+      logger.error("unable to retrieve Iam Dynamo Table Name from config - check that table name is present in security-hq.conf in S3")
+      "error"
```

Comment:
If there is no table name in SHQ's config, the "error" string will be used to initialise the Dynamo class. Then, a `try catch` is used when we call the `put` method in `Dynamo.scala` which will catch the error and log out the error message. These two log lines combined should be good information if we need to debug this particular problem in the future.

Reply:
It's generally not the best idea to use a String to contain either an error or the result - probably better to keep the value as an option and deal with the option later